microsoft-365/security/office-365-security/step-by-step-guides/Track and respond to emerging threats with campaigns.md

```diff
@@ -0,0 +1,78 @@
+1. Navigate to the campaign page: https://security.microsoft.com/campaigns 
+1. Select the campaign name that you would like to investigate. 
+1. Upon the flyout opening, select **“Download threat report.”**
+1. Open the threat report and it will provide more information surrounding the campaign. The information in the report includes: 
+- **Executive summary:** High level summary of the type of campaign and the number of users targeted in your organization. 
+- **Analysis:** Timeline chart of when the campaign started, the count of messages targetting your organization, and the destination and verdicts of the messages. 
+- **Attack origin:** Top sending IP addresses and domains with a count of messages that were delivered to inboxes in your organization. This allows you to investigate who is targeting your organization. 
+-	**Email template and payload:** The subject line of the emails that were part of the campaign and URLs (and their frequency) present as part of the campaign.
+-	**Recommendations:** Recommendations for next steps to remediate messages.
+
+## Investigate Inboxed messages part of a campaign
+
+1.	Navigate to the campaign page: security.microsoft.com/campaigns.
+1.	Scroll through the list of campaigns in the **Details view**, below the graph.
+1.	**Select** the campaign name you want to investigate. If the campaign has a click count more than zero, that indicates that a user in your organization clicked on a URL or downloaded a file from the email.
+1.	The **campaign flyout** displays more information about the campaign, the graph displays a timeline of the campaign from campaign start to end date & the horizonal flow diagram displays the stages of the campaign from its origin, the verdict, and the current location of the messages.
+1.	Below the flow diagram, selecting the **URL clicks** tab displays information regarding the click. Here you can see the user that clicks on a URL, if the user is tagged as a priority account user, the URL itself, and the time of click. 
+1.	If you want to learn more about the inboxed and clicked messages. Click on **Explore messages** > **Inboxed messages**. A new tab will open and navigate to Threat Explorer. 
+1.	In the **details view** of Explorer you can reference **Latest delivery** to determine if a message is still in the Inbox or was moved into quarantine by system ZAP. _To get more details about the specific message, select the message. The fly out provides extra information. Upon selecting the Open email entity page on the top left of the flyout, a new tab will open and give you further information about the message._
+1.	If you would like to take an action and move the messages out of the inbox, you can **select the message** and click on **Message actions** > **Move to junk folder**. This will ensure your user doesn’t continue to interact with the malicious message that could result in a potential breach. 
```

Comment:
```suggestion
1. The campaign flyout displays more information about the campaign, the graph displays a timeline of the campaign from campaign start to end date, and the horizontal flow diagram displays the stages of the campaign from its origin, the verdict, and the current location of the messages.
1. Below the flow diagram, select the **URL clicks** tab to display information regarding the click. Here you can see the user that clicked on a URL, if the user is tagged as a priority account user, the URL itself, and the time of click. 
1. If you want to learn more about the inboxed and clicked messages, select **Explore messages** > **Inboxed messages**. A new tab will open and navigate to Threat Explorer. 
1. In the **details view** of Explorer you can reference **Latest delivery** to determine if a message is still in the inbox or was moved into quarantine by system ZAP. _To get more details about the specific message, select the message. The flyout provides extra information. Upon selecting the **Open email entity page** on the top left of the flyout, a new tab will open and give you further information about the message._
1.	If you would like to take an action and move the messages out of the inbox, you can select the message and then select **Message actions** > **Move to junk folder**. This will ensure your user doesn’t continue to interact with the malicious message that could result in a potential breach. 
```